addons/website_slides_survey/views/survey_survey_views.xml

```diff
@@ -25,4 +25,23 @@
             </p>
         </field>
     </record>
-</odoo>
\ No newline at end of file
+
+    <record id="survey_survey_view_form" model="ir.ui.view">
+        <field name="name">survey.survey.view.form.inherit.website_slides</field>
+        <field name="model">survey.survey</field>
+        <field name="inherit_id" ref="survey.survey_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//button[@name='action_survey_user_input_certified']" position="before">
+                <button name="action_survey_view_slide_channels"
+                    type="object"
+                    class="oe_stat_button"
+                    attrs="{'invisible': [('slide_channel_count', '=', 0)]}"
+                    icon="fa-graduation-cap"
+                    groups="website_slides.group_website_slides_officer">
+                    <field string="Courses" name="slide_channel_count" widget="statinfo"/>
+                </button>
+            </xpath>
+        </field>
+    </record>
+</odoo>
+
```

Comment:
Now there are 2 carriage returns, 1 is enough ^^

Reply:
Sorry about that! Fixed it!